config/proto_conversions.go

```diff
@@ -164,23 +169,26 @@ func ComponentConfigFromProto(protoConf *pb.ComponentConfig) (*resource.Config,
 		return nil, err
 	}
 
-	level := logging.INFO
+	var logConfig *resource.LogConfig
 	if protoConf.GetLogConfiguration() != nil {
-		if level, err = logging.LevelFromString(protoConf.GetLogConfiguration().Level); err != nil {
+		level, err := logging.LevelFromString(protoConf.GetLogConfiguration().Level)
+		if err != nil {
 			// Don't fail configuration due to a malformed log level.
 			level = logging.INFO
 			logging.Global().Warnw(
 				"Invalid log level.", "name", protoConf.GetName(), "log_level", protoConf.GetLogConfiguration().Level, "error", err)
 		}
+		logConfig = &resource.LogConfig{Level: level}
 	}
+
 	componentConf := resource.Config{
 		Name:                      protoConf.GetName(),
 		API:                       api,
 		Model:                     model,
 		Attributes:                attrs,
 		DependsOn:                 protoConf.GetDependsOn(),
 		AssociatedResourceConfigs: serviceConfigs,
-		LogConfiguration:          resource.LogConfig{Level: level},
+		LogConfiguration:          logConfig,
```

Comment:
So this will be `nil` if no `log_configuration` was supplied by the user? What is the expected log level of a resource with no `log_configuration`? Whatever `rdk.resource_manager` is set to?

Reply:
yes exactly!